packages/react-components/react-drawer/src/components/DrawerOverlay/useDrawerOverlayStyles.styles.ts

```diff
@@ -2,9 +2,11 @@ import { makeStyles, mergeClasses } from '@griffel/react';
 import type { DrawerOverlaySlots, DrawerOverlayState } from './DrawerOverlay.types';
 import type { SlotClassNames } from '@fluentui/react-utilities';
 import { useDrawerBaseStyles } from '../../util/useDrawerBaseStyles.styles';
+import * as React from 'react';
 
 export const drawerOverlayClassNames: SlotClassNames<DrawerOverlaySlots> = {
   root: 'fui-DrawerOverlay',
+  backdrop: 'fui-DrawerOverlay__backdrop',
```

Comment:
what is this `backdrop`?

Reply:
This is the background scrim element for the Drawer. It uses the same API as the Dialog component.